### ISSUE TITLE
Missing webpack context rules for requiring lazy messages file

### ISSUE THREAD

**Issue by existe-deja** [nuxt-modules/i18n#1068]
### Version
nuxt-i18n: 6.20.1
nuxt: 2.15.0

#### [mode](https://nuxtjs.org/guides/configuration-glossary/configuration-mode): 
 - [X] universal
 - [ ] spa

### Nuxt-i18n configuration
  i18n: {
    locales: [{ code: "fr", file: "fr.js" }],
    langDir: "lang/lazyDir",
    defaultLocale: "fr",
    lazy: true
  }

### Reproduction Link
https://codesandbox.io/s/romantic-hamilton-o1ijf?file=/lang/README.md

### Steps to reproduce
Insert a .md file inside the folder where lazy files are loaded (aka langDir property)

### What is Expected?
No webpack warnings for unhandled files.

### What is actually happening?
Webpack wants me to handle the md files. It throws this warning
```
./lang/lazyDir/README.md 1:5
Module parse failed: Unexpected token (1:5)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
```

### Thoughts
Maybe it's because of require context who parse too much file extensions.
A quick fix is to keep the langDir as light as possible without md files, and require messages from another folder.

**Comment by rchl** [comment]
I'll only try to load files with `js`, `ts`, and `json` extensions. Let me know if you can think of any other that should be included.

**Comment by existe-deja** [comment]
I think this is fine. Well done for reactivity :clap:

**Comment by rchl** [comment]
Released the fix in https://github.com/nuxt-community/i18n-module/releases/tag/v6.20.2

**Comment by kedrzu** [comment]
Would appreciate adding YAML files to the list (`yml`, `yaml`).

**Comment by kedrzu** [comment]
Up until now, the module was able to handle any file, that webpack would recognize. So changing this behavior is really a breaking change, not just a fix - for example breaking my application as soon as I bumped the library.

Also, it maybe would be nice to add the possibility to define custom file types in the config.

**Comment by rchl** [comment]
Sorry about that. Addressed with 01f92ae72ba2a37cda2be119f618751ab7dbf20b and released in https://github.com/nuxt-community/i18n-module/releases/tag/v6.20.3

> Also, it maybe would be nice to add the possibility to define custom file types in the config.

I will consider that if there is really a need for it (that is if someone complains again).
